test_unstructured_ingest/expected-structured-output/elasticsearch/movies/0-57554198.json

```diff
@@ -7,8 +7,7 @@
           "document_id": "0",
           "index_name": "movies",
           "url": "http://localhost:9200"
-        },
-        "version": "1"
```

Comment:
What was this version 1? and why did we remove it. Is this how we were going to incrementally ingest docs?

Reply:
This was a ramification of getting a specific record by id rather than a general scan for a lot of ids at once in a batch call. This value doesn't exist on the response from that call.

Reply:
Actually, that might be a configuration I can set to get that back. Let me take a look.

Reply:
Added it back in.